flink-java/src/main/java/org/apache/flink/api/java/utils/DataSetUtils.java

```diff
@@ -412,8 +412,7 @@ public TupleSummaryAggregator<R> reduce(
      *
      * @return A ChecksumHashCode that represents the count and checksum of elements in the data
      *     set.
-     * @deprecated replaced with {@code org.apache.flink.graph.asm.dataset.ChecksumHashCode} in
-     *     Gelly
+     * @deprecated replaced with {@code org.apache.flink.api.java.ChecksumHashCode}
```

Comment:
`ChecksumHashCode` is not really a replacement.
Let's just say this method will simply be removed at some point.